### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ const keyPair = libsodium.crypto_sign_keypair();
 ### Requirements
 
 - [Node.js](https://nodejs.org/)
-- Rust (use [rustup](https://rustup.rs/))
-- [Neon](https://github.com/neon-bindings/neon)
+- [Rust](https://www.rust-lang.org/install.html)
+- For Windows also [windows-build-tools](https://www.npmjs.com/package/windows-build-tools)
```
Comment (bennyn) [inline README.md]: The command `npm install -g --production windows-build-tools` should be mentioned here.

File: README.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ const keyPair = libsodium.crypto_sign_keypair();
 ### Requirements
 
 - [Node.js](https://nodejs.org/)
-- Rust (use [rustup](https://rustup.rs/))
-- [Neon](https://github.com/neon-bindings/neon)
```
Comment (bennyn) [inline README.md]: No need for `npm install -g neon-cli` anymore?

Comment (ffflorian) [inline README.md]: No, npm will find it in `node_modules`.

File: README.md (hunk 1 above)
Comment (ffflorian) [inline README.md]: The command is visible in the npm package :slightly_smiling_face:

**Pull request merged**